about.html

```diff
@@ -0,0 +1,120 @@
+            <ul class="second-nav-content ">
+                <li><a href="" class="links"></a>About</li>
+                <li><a href="" class="links"></a>Program </li>
+                <li><a href="" class="links"></a>Join</li>
+                <li><a href="" class="links"></a>Sponsor</li>
+                <li><a href="" class="links"></a>News</li>
+                <button class="delete"  onclick="closeMobileMenu()">
+                </button>
+                <button class="nav-button"><a href=""></a>FIFA compaign</button>
+            </ul>
+        </nav>
+        <button onclick="openMobileMenu()" class="toolbar" >
+            <svg xmlns="http://www.w3.org/2000/svg" class="icon-menu" width="44" height="44" viewBox="0 0 24 24"
+              stroke-width="1.5" stroke="#272a32" fill="none" stroke-linecap="round" stroke-linejoin="round">
+              <path stroke="none" d="M0 0h24v24H0z" fill="none" />
+              <line x1="4" y1="6" x2="20" y2="6" />
+              <line x1="4" y1="12" x2="20" y2="12" />
+              <line x1="4" y1="18" x2="20" y2="18" />
+            </svg>
+          </button>
+    </header>
+    <section class="about about-1">
+        <div>
+            <h1>WELCOME TO FIFA WORLD <br> CUP QATAR 2022 </h1>
+        </div>
+        <div class="headlin-description about-description">
+            <p>
+                The new FIFA is modernising football to be global, accessible and inclusive in all aspects. Not just on one or two continents, but everywhere.
+
+                Under our vision to make football truly global, we will help develop football everywhere so that there are at least 50 national teams and 50 clubs from all continents that can compete at a top competitive level.            </p>
+        </div>
+        <div class="contact">
+            <p>
+                Please contact us via Email for any further questions about FIFA &
+                QATAR world cup 2022!
+              </p>
+              <a href="#">worldcup@FIFA.com </a>
+        </div>
+    </section>
+   <section class="main-about">
+        <h2>ALL FIFA</h2>
```

Comment:
- [ ] This element has a tiny orange horizontal line beneath it in the Behance template. Kindly make your app look identical to it. 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/205458413-9ebeadf1-ac90-40c8-b4ba-5a39d9901386.png)

### Behance Template

![image](https://user-images.githubusercontent.com/77106962/205458435-2b33d73c-d432-40af-9a46-5bc1e8b483fb.png)